package.json

```diff
@@ -21,14 +25,14 @@
     "discord.js-plugins": "git+https://github.com/NbOpposite/discord.js-plugins.git",
     "emojis-list": "^2.1.0",
     "jsdom": "^13.2.0",
+    "libxmljs": "0.19.5",
```

Comment:
This is not a direct dependency of this project, and is already listed as a dependency [here](https://github.com/bpkennedy/wolfram-alpha/blob/302d0debe337b8c329f584329e8b79e7a185a6cb/package.json#L23)

Reply:
Ah right - I forgot that I updated it in my form of WA.  I will take that out.  And also put back the node-opus.  Although node-opus was doing a lot of complaining.  Maybe it was related to the libxmljs thing though....